app-frontend/src/app/app.js

```diff
@@ -0,0 +1,38 @@
+import angular from 'angular';
```

Comment:
2 Application Settings should be added:
 1. [strict di](https://github.com/johnpapa/angular-styleguide/tree/master/a1#ng-annotate) (see end of section) to help protect against minification issue
 2. Turn off [debug info](https://medium.com/swlh/improving-angular-performance-with-1-line-of-code-a1fb814a6476) -- arguable to keep it on in development, but for now I think just turning it off is the safest bet